irony-completion.el

```diff
@@ -252,11 +268,12 @@ A candidate is composed of the following elements:
  2. The [result-]type of the candidate, if any.
  3. If non-nil, contains the Doxygen brief documentation of the
     candidate.
- 4. The signature of the candidate excluding the result-type
+ 4. The accessiblity of the candidate.
```

Comment:
Any reason to put this to the 4th position?
The `irony-completion-candidates` function is publicly documented the candidate "objects" are supposed to be "ABI stable" and extensible (by adding new fields).

Nit: could add a reference to `irony-completion-accessibility-filter` which lists the possible values.